tox.ini

```diff
@@ -2,8 +2,9 @@
 skipsdist = True
 usedevelop = True
 envlist =
-    py{37,38,39}-dj{30,31,32}-wt{215}
-    py{37,38,39}-dj{32,40}-wt{216,30}
+    py{37,38,39,310}-dj{32}-wt{215,216}
+    py{38,39,310}-dj{32,40}-wt{216,30,40}
+    py{38,39,310,311}-dj{41}-wt{40}
```

Comment:
When adding Wagtail 4.1 the tests are failing but the CMS interface seems to be working OK.
I can look at this when we need it for Wagtail 4.1